optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/ProblemBenchmarkResult.java

```diff
@@ -79,6 +78,9 @@
     private Long variableCount = null;
     private Long problemScale = null;
 
+    @XStreamOmitField // loaded lazily from singleBenchmarkResults
+    private Boolean hasNonDefaultSubSingleCount = null;
```

Comment:
Naming. For what is this boolean used?
If it is used to determine whether or not something should be shown because showing just 1 subsingle is pointless,
then it should not be named "...NonDefault..." because the default value doesn't matter then (as it's a coincidence that the default is 1 too), instead a name like "subSingleCountIsMultiple" or "subSingleCountAtLeastTwo" or maybe track an int such as "int subSingleCount" or "int maximumSubSingleCount" (in case there are multiple singles per problem and they could have a different subSingleCount such as during an Aggregator).